cdap-ui/app/hydrator/controllers/create/partials/nodeconfig-ctrl.js

```diff
@@ -466,6 +467,10 @@ class HydratorPlusPlusNodeConfigCtrl {
         vm.propertyErrors = {};
         vm.inputSchemaErrors = {};
         vm.outputSchemaErrors = {};
+        // Do not show success validation message for validation via get schema.
+        if (noSuccessIndicator) {
+          vm.errorCount = null;
```

Comment:
Why is errorCount being set to null here, as opposed to undefined? Might be good to update the comment on line 458 to add the null case as well.